### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by afinne** [gangstead/atom-incremental-search#35]
[Enter steps to reproduce below:]

1. Cmd-I to enter incremental search
2. Type something
3. Press Esc to try and close the Incremental Search

**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [incremental-search](https://github.com/gangstead/atom-incremental-search) package, v5.0.1

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/afinne/.atom/packages/incremental-search/lib/search-model.coffee:168

TypeError: undefined is not a function
  at SearchModel.module.exports.SearchModel.cleanup (/Users/afinne/.atom/packages/incremental-search/lib/search-model.coffee:168:32)
  at SearchModel.module.exports.SearchModel.cancelSearch (/Users/afinne/.atom/packages/incremental-search/lib/search-model.coffee:151:6)
  at InputView.module.exports.InputView.cancelSearch (/Users/afinne/.atom/packages/incremental-search/lib/input-view.coffee:147:18)
  at space-pen-div.subscriptions.add.atom.commands.add.core:cancel (/Users/afinne/.atom/packages/incremental-search/lib/input-view.coffee:37:26)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
  2x -1:21.7.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -1:20.7.0 core:select-down (atom-text-editor.editor.is-focused)
     -1:20.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:19.9.0 core:move-down (atom-text-editor.editor.is-focused)
     -1:19.8.0 core:move-left (atom-text-editor.editor.is-focused)
     -1:04.6.0 core:move-right (atom-text-editor.editor.is-focused.autocomplete-active)
     -1:04.3.0 core:move-down (atom-text-editor.editor.is-focused)
  2x -1:04.2.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:17.4.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:15.4.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:15.4.0 git-diff-details:close-git-diff-details (atom-text-editor.editor.mini.is-focused)
     -0:15.4.0 core:cancel (atom-text-editor.editor.mini.is-focused)
  2x -0:14.7.0 incremental-search:forward (atom-text-editor.editor.is-focused)
     -0:01.8.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:01.8.0 git-diff-details:close-git-diff-details (atom-text-editor.editor.mini.is-focused)
     -0:01.8.0 core:cancel (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "tree-view-open-files",
      "language-mustache",
      "atom-handlebars",
      "better-handlebars",
      "compare-files"
    ],
    "destroyEmptyPanes": false
  }
}
```

### Installed Packages

```coffee
# User
Termrk, v0.1.17
file-icons, v1.5.7
git-control, v0.3.0
git-diff-details, v0.18.0
incremental-search, v5.0.1
language-ember-htmlbars, v0.3.0
linter, v0.12.6
linter-coffeelint, v0.3.2
linter-handlebars, v0.1.1
linter-rubocop, v0.2.7
linter-ruby, v0.1.6
minimap, v4.9.4
pretty-json, v0.3.3
project-manager, v1.15.10
tag, v0.3.0

# Dev
No dev packages
```

**Comment by drogar** [comment]
+1 - getting the same issue.

**Comment by theres-waldo** [comment]
I'm seeing this issue as well.

**Comment by o-jill** [comment]
+1

**Comment by lorencs** [comment]
+1

**Comment by exalted** [comment]
+1

**Comment by chadkirby** [comment]
+1

**Comment by Jaswetz** [comment]
+1

**Comment by rogerpence** [comment]
+1

**Comment by trishume** [comment]
+1

**Comment by allen-zeng** [comment]
+1

**Comment by afinne** [comment]
If anyone wants to, you could try out the change in my PR above.

**Comment by kedashoe** [comment]
Hi @afinne , thanks for your work on this! I tried running your fork and am still getting an error. Steps to reproduce are the same, I'm running Atom 1.0.2 on Ubuntu 14.04

```
Uncaught TypeError: undefined is not a function
/home/kevin/src/a/atom-incremental-search/lib/search-model.coffee:171

TypeError: undefined is not a function
    at SearchModel.module.exports.SearchModel.cleanup (/home/kevin/src/a/atom-incremental-search/lib/search-model.coffee:164:38)
    at SearchModel.module.exports.SearchModel.cancelSearch (/home/kevin/src/a/atom-incremental-search/lib/search-model.coffee:147:19)
    at InputView.module.exports.InputView.cancelSearch (/home/kevin/src/a/atom-incremental-search/lib/input-view.coffee:238:24)
    at space-pen-div.subscriptions.add.atom.commands.add.core:cancel (/home/kevin/src/a/atom-incremental-search/lib/input-view.coffee:96:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
    at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:177:20)

**Comment by rpietro** [comment]
+1

**Comment by aullman** [comment]
+1 I'm seeing this issue too.

**Comment by nicothed** [comment]
+1

**Comment by donovanlange** [comment]
+1

**Comment by manteca-coloraa** [comment]
Hi, I also tried the fix and it didn't work for me. Same as @kedashoe.

**Comment by klingel** [comment]
+1

**Comment by vigoo** [comment]
+1

**Comment by ChadMartinson** [comment]
+1

**Comment by tujoworker** [comment]
+1

**Comment by gangstead** [comment]
Published `v5.0.2`
